### ISSUE TITLE
Lost the L for local branches

### ISSUE THREAD

**Issue by hypergig** [magicmonty/bash-git-prompt#147]
Looks like something in git 2.5 is incompatible with the Local branch check. I no longer see the L in the prompt.

**Comment by hypergig** [comment]
More info:

I did upgrade bash-git-prompt at the same time I upgraded git.. I just brew upgrade.  It may not be git after all.  Doing some more investigation.

**Comment by ogr3** [comment]
Found a bug in gitstatus.sh - the check for upstream branch is language dependent - i.e. if you have another language set than English the test will not work.

I'll have a pull request ready for review soon.

**Comment by ogr3** [comment]
If this is your problem, it will be fixed by #173

**Comment by hypergig** [comment]
I am using english... but I would be happy to test it when its ready

**Comment by easterwood** [comment]
Hmm, I use git version 2.5.2.windows.2 and if I create a local branch the value for ```GIT_PROMPT_SYMBOLS_NO_REMOTE_TRACKING``` is displayed. The  ```GIT_PROMPT_SYMBOLS_NO_REMOTE_TRACKING``` is the variable which contains the ```L``` symbol. Maybe it is overridden by a custom template.

**Comment by ogr3** [comment]
It's a bug in the existing code as well. They recently changed it to test for

```
fatal: no upstream
```

But on my Ubuntu with English language set i get
```
fatal: No upstream configured for branch 'nisse'
```

So this too is fixed by merging #173, which makes this test language and git version independent.

**Comment by ogr3** [comment]
This is now fixed by #170, which hasn't been accepted yet into master.

**Comment by magicmonty** [comment]
Should be fixed now, as #170 is now integrated into master

**Comment by hypergig** [comment]
Thank you all! I will brew upgrade, see what's what.
